pylearn2/train_extensions/tests/test_roc_auc.py

```diff
@@ -112,7 +112,7 @@ def test_roc_auc_one_vs_one():
                     max_epochs: 1,
                 },
                 !obj:pylearn2.termination_criteria.MonitorBased {
-                    channel_name: train_roc_auc-0vX,
+                    channel_name: train_roc_auc_0vX,
```

Comment:
This seems to trigger the following error in Travis: `KeyError: "'train_roc_auc_0vX'`